media/windows/calendar-reservations/calendar-reservations.py

```diff
@@ -179,7 +412,7 @@ def process_events(service, calendar, log):
         events_to_respond_to = check_response_status(events, calendar, log)
 
         #checks who the organizer of each event is and accepts or declines the event
-        respond_to_events(events_to_respond_to, service, calendar, log)
+        respond_to_events(events_to_respond_to, events, service, calendar, log)
```

Comment:
It may be simpler to take 2 passes through all the events that we get back from Google:

1. First pass: reject all non-acceptable-domain events and actually delete them from our list.
2. Second pass: we know that all events left in the list are "good" events (i.e., they are valid for comparison for time conflicts).  So now search this list for events to which we need to respond, and when we do the check for time conflicts, _any_ event that conflicts is a valid conflict (because we know that only "good" events are left).